lib/git-process.ts

```diff
@@ -163,23 +163,32 @@ export class GitProcess {
       }
 
       const spawnedProcess = execFile(gitLocation, args, execOptions, function(
-        err: ErrorWithCode,
+        err: Error | null,
         stdout,
         stderr
       ) {
-        const code = err ? err.code : 0
+        if (err === null) {
```

Comment:
Strict comparison with `null` always makes me nervous, but if we're confident this won't be `undefined` it's ok.

Reply:
I can relax this to `if (!err)`